sleepio/community-app-web
# PG-1236 Community Email Subscription Settings

**Reminder to create the PR against this repository, and not the base fork of `rafalp/Misago`.**

### JIRA Ticket

https://bighealth.atlassian.net/browse/PG-1236

### Problem

As a user, I would like the emails I receive from community notifying me of updates to threads I’m subscribed to to contain a link that allows me to easily manage my Community email subscription settings, so I can opt-in and out of email notifications quickly and easily

### Solution

* We have added a new component to the /subscribed page in Community to allow bulk sub, sub-email, and unsub
* We have updated the subscription email templates to include a link to the subscription page
* We have updated the login redirect URL generation to follow the url formatting to allow redirection back to Community

### Test Plan

#### Unit tests + Integration Tests

* updated test_auth_middleware.py to validate new redirect point

#### Manual Tests

* We won't actually see redirection back to /subscriptions when a user logs in after clicking the link in their email, we should instead validate that the redirect URL is structured correctly. See https://bighealth.atlassian.net/browse/PG-1501 and https://bighealth.atlassian.net/browse/ENR-31

### Rollout


#### Related PRs
